packages/core/src/stylable-value-parsers.ts

```diff
@@ -133,53 +120,6 @@ export const SBTypesParsers = {
             types,
         };
     },
-    '-st-mixin'(
-        mixinNode: postcss.Declaration,
-        strategy: (type: string) => 'named' | 'args',
-        diagnostics?: Diagnostics,
-        emitStrategyDiagnostics = true
-    ) {
-        const ast = postcssValueParser(mixinNode.value);
-        const mixins: Array<MixinValue> = [];
-
-        function reportWarning(message: string, options?: { word: string }) {
-            if (emitStrategyDiagnostics) {
-                diagnostics?.warn(mixinNode, message, options);
-            }
-        }
-
-        ast.nodes.forEach((node) => {
-            if (node.type === 'function') {
-                mixins.push({
-                    type: node.value,
-                    options: strategies[strategy(node.value)](node, reportWarning),
-                    valueNode: node,
-                    originDecl: mixinNode,
-                });
-            } else if (node.type === 'word') {
-                mixins.push({
-                    type: node.value,
-                    options: strategy(node.value) === 'named' ? {} : [],
-                    valueNode: node,
-                    originDecl: mixinNode,
-                });
-            } else if (node.type === 'string') {
-                diagnostics?.error(mixinNode, valueParserWarnings.VALUE_CANNOT_BE_STRING(), {
-                    word: mixinNode.value,
-                });
-            }
-        });
-
-        return mixins;
-    },
-    '-st-partial-mixin'(
-        mixinNode: postcss.Declaration,
-        strategy: (type: string) => 'named' | 'args',
-        diagnostics?: Diagnostics
-    ) {
-        return SBTypesParsers['-st-mixin'](mixinNode, strategy, diagnostics).map((mixin) => {
-            mixin.partial = true;
-            return mixin;
-        });
-    },
+    '-st-mixin': parseStMixin,
```

Comment:
these `SBTypesParsers` are not used outside of Stylable so we can eliminate them and the maintenance of them